src/components/Form/Readme.mdx

```diff
@@ -198,6 +308,56 @@ Invoke `getSelectStyling` by passing it the current theme object, to receive con
   </ThemeConsumer>
 </Playground>
 
+## RangeSlider
+
+<Props of={RangeSlider} />
+Props table might not render due to a [bug in docz](https://github.com/pedronauck/docz/issues/777)
+
+<Playground>
+  {() => {
+    const [formData, setFormData] = useState({});
+    const [lastEvent, setLastEvent] = useState({});
+    const onChange = (e) => {
+      setLastEvent(e.target);
+      const { name, value } = e.target;
+      setFormData({ ...formData, [name]: value })
+    }
+    return (
+      <Flex>
+        <Box margin="base" padding="base" maxWidth="400px">
+          <Box bg="neutral.lighter">
+            <RangeSlider
+              name="engagement"
+              value={formData.engagement}
+              onChange={onChange}
+              label="Engagement score?"
```

Comment:
label doesn't render.

Reply:
Yes, I think we may just remove the label there all together, it looks weird on a `RangeSlider`

Reply:
it does, kinda, guess that's why I have huge titles over it for TR... :)